src/backends/torch/torchdataset.h

```diff
@@ -89,6 +89,10 @@ namespace dd
     bool _test = false;                 /**< whether a test set */
     TorchImgRandAugCV _img_rand_aug_cv; /**< image data augmentation policy. */
 
+    // test control
+    int _test_crop_samples
```

Comment:
This is supposed to be only related to test, but in the code this actually controls the number of sample per image. The name could be more generic?

Reply:
OK, changed to `n_crop_samples`, but the API option remains `"test_crop_samples"` because it only affects test samples.